CONTRIBUTING.md

```diff
@@ -4,22 +4,38 @@
 
 If you would like to contribute enhancements or fixes, please do the following:
 
-1.  Fork the package repository.
-2.  Run `npm install` to setup all dependencies.
-3.  Hack on a separate topic branch created from the latest `master`.
-4.  Check for lint errors with `npm run lint` or use `linter-eslint` within Atom.
-5.  Run the package specs.
-6.  Commit the changes and push the topic branch to your fork.
-7.  Make a pull request to the main repo.
-8.  Welcome to the club!
+1. Fork the package repository and clone your fork to local file.
+
+1. Run `npm install` in the project to setup all dependencies.
```

Comment:
_Technically_ it should be `apm install`, since then you get the packages installed how Atom will be installing them, but I'm perfectly fine with `npm install` since it will generate a _much_ better tree than the current `npm@3.10.10` bundled in `apm`.

Reply:
Also I notice you've switched to `1.` for all the numbering. I'm not really _against_ that as it does make diffs easier to read, but later in the document another list is back to `n.` numbering. Note that `remark` under the default config will yell at you for this style, I never got around to adding a configuration here for that so I'm not going to enforce either method 😉.

Reply:
`npm` was already in the text, I didn't put it there. It's what I used and it worked for me. If you think `apm` is a better instruction to give I can change it.

Not sure how I missed some of the numbering. My intention at least was to change them all. I agree it will give nicer on diffs, and it's also easier to edit. You can move things around, insert or delete, etc. without worrying about renumbering.

Technically I could have just inserted 1 for my new entries and it would have worked itself out, but that seems pretty gross to me. All 1s is a lot less icky.

I don't know anything about `remark`, but I'm happy to change it back if you want, or just finish what I started.

Reply:
I'd say just change them all to `1.`, I slightly prefer that. At some point I'll add `remark` here and we can fix it then if we want to or _enforce_ it then 😉.